base_layer/core/src/mempool/mempool_storage.rs

```diff
@@ -88,10 +93,6 @@ impl MempoolStorage {
                 );
                 let timer = Instant::now();
```

Comment:
Unrelated: It looks like this timer shouldnt be here - there is a timer above which includes validation time.

Reply:
No this is correct, its a new timer. 
Just above this, we print the validation results. This timer is just for the db insertion.